backend/src/main/resources/logback.xml

```diff
@@ -2,7 +2,9 @@
 
   <appender class="ch.qos.logback.core.ConsoleAppender" name="STDOUT">
     <encoder>
-      <pattern>%cyan(%d{HH:mm:ss.SSS}) %gray([%thread]) %highlight(%-5level) %magenta(%logger{36}) - %msg%n</pattern>
+      <pattern>%cyan(%d{HH:mm:ss.SSS}) %gray([%thread]) %highlight(%-5level) %magenta(%logger{36}) -
+        %msg%n
+      </pattern>
```

Comment:
I'm not a fan of newlines in the log, but I will probably change the log format completly in the next time so this is fine for now

Reply:
I would also leave it like this for now